pages/common/http.md

```diff
@@ -11,9 +11,9 @@
 
 `http --print {{H|B|h|b|m|Hh|Hhb|...}} {{https://example.com}}`
 
-- Specify the HTTP method when sending a request:
+- Specify the HTTP method when sending a request and use a proxy to intercept the request:
 
-`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} {{https://example.com}}`
+`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} --proxy {{http|https|...}}:{{http://localhost:8080|socks5://localhost:9050|...}} {{https://example.com}}`
```

Comment:
```suggestion
`http {{GET|POST|HEAD|PUT|PATCH|DELETE|...}} --proxy {{http|socks5|...}}:{{http://localhost:8080|socks5://localhost:9050|...}} {{https://example.com}}`
```

Reply:
I had the same confusion! This is actually the protocol for the **target url**. If the target is http://example.com, the protocol is `http`. If https://example.com, then `https`. I am not aware of any target url like socks5://example.com.